lib/serverless.js

```diff
@@ -0,0 +1,8 @@
+const { buildMessage, getPullRequests } = require('./helpers')
+const { notifyMsTeams } = require('./ms-teams')
+
+module.exports.run = async event => {
+  const prs = await getPullRequests()
+  const message = await buildMessage(prs)
+  await notifyMsTeams(message)
```

Comment:
Probably should note somewhere in README or something that serverless is only configured for Teams. (maybe include a slack TODO here?)